devops/dependencies.json

```diff
@@ -18,6 +18,12 @@
       "url": "https://github.com/intel/cm-compiler/releases/tag/cmclang-1.0.144",
       "root": "{DEPS_ROOT}/opencl/runtime/linux/oclgpu"
     },
+    "level_zero": {
+      "github_tag": "v1.8.8",
+      "version": "v1.8.8",
+      "url": "https://github.com/oneapi-src/level-zero/releases/tag/v1.8.8",
+      "root": "{DEPS_ROOT}/opencl/runtime/linux/oclgpu"
+    },
```

Comment:
https://github.com/oneapi-src/level-zero/releases/

Tested Platforms:
[Ubuntu 18.04](http://releases.ubuntu.com/18.04.4/)
[Ubuntu 20.04](http://releases.ubuntu.com/20.04/)
Windows 11

Hm... Ubuntu 22.04 is not in the list.

Reply:
That's probably just an old documentation. There are no separate packages per each Ubuntu version. I assume 18.04 works everywhere. In any case we will test.